I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/store.rpy", line 35, in script
    
  File "renpy/common/000statements.rpy", line 531, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
  File "game/store.rpy", line 38, in execute
    
  File "game/store.rpy", line 38, in execute
    
  File "game/store.rpy", line 53, in execute
    text "Clients: [clients_girl]" ypos 100 xpos
  File "game/store.rpy", line 63, in execute
    $xcord = 500
  File "game/store.rpy", line 63, in execute
    $xcord = 500
  File "game/store.rpy", line 66, in execute
    if state[0] == 1:#comprar
  File "game/store.rpy", line 67, in execute
    $item_to_choose=[]
  File "game/store.rpy", line 73, in execute
    $mesa= item_picked[1]
  File "game/store.rpy", line 73, in <module>
    $mesa= item_picked[1]
TypeError: 'int' object has no attribute '__getitem__'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/store.rpy", line 35, in script
    
  File "D:\Gamedev\renpy-7.3.5-sdk\renpy\ast.py", line 1949, in execute
    self.call("execute")
  File "D:\Gamedev\renpy-7.3.5-sdk\renpy\ast.py", line 1937, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "D:\Gamedev\renpy-7.3.5-sdk\renpy\statements.py", line 277, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 531, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
  File "D:\Gamedev\renpy-7.3.5-sdk\renpy\exports.py", line 2905, in call_screen
    rv = renpy.ui.interact(mouse="screen", type="screen", roll_forward=roll_forward)
  File "D:\Gamedev\renpy-7.3.5-sdk\renpy\ui.py", line 297, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "D:\Gamedev\renpy-7.3.5-sdk\renpy\display\core.py", line 2702, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "D:\Gamedev\renpy-7.3.5-sdk\renpy\display\core.py", line 3094, in interact_core
    root_widget.visit_all(lambda i : i.per_interact())
  File "D:\Gamedev\renpy-7.3.5-sdk\renpy\display\core.py", line 541, in visit_all
    d.visit_all(callback, seen)
  File "D:\Gamedev\renpy-7.3.5-sdk\renpy\display\core.py", line 541, in visit_all
    d.visit_all(callback, seen)
  File "D:\Gamedev\renpy-7.3.5-sdk\renpy\display\core.py", line 541, in visit_all
    d.visit_all(callback, seen)
  File "D:\Gamedev\renpy-7.3.5-sdk\renpy\display\screen.py", line 430, in visit_all
    callback(self)
  File "D:\Gamedev\renpy-7.3.5-sdk\renpy\display\core.py", line 3094, in <lambda>
    root_widget.visit_all(lambda i : i.per_interact())
  File "D:\Gamedev\renpy-7.3.5-sdk\renpy\display\screen.py", line 440, in per_interact
    self.update()
  File "D:\Gamedev\renpy-7.3.5-sdk\renpy\display\screen.py", line 625, in update
    self.screen.function(**self.scope)
  File "game/store.rpy", line 38, in execute
    
  File "game/store.rpy", line 38, in execute
    
  File "game/store.rpy", line 53, in execute
    text "Clients: [clients_girl]" ypos 100 xpos
  File "game/store.rpy", line 63, in execute
    $xcord = 500
  File "game/store.rpy", line 63, in execute
    $xcord = 500
  File "game/store.rpy", line 66, in execute
    if state[0] == 1:#comprar
  File "game/store.rpy", line 67, in execute
    $item_to_choose=[]
  File "game/store.rpy", line 73, in execute
    $mesa= item_picked[1]
  File "game/store.rpy", line 73, in <module>
    $mesa= item_picked[1]
TypeError: 'int' object has no attribute '__getitem__'

Windows-7-6.1.7601-SP1
Ren'Py 7.3.5.606
aTry 1.0
Wed Jun 03 18:50:13 2020
